plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/write/TestDataWritableWriter.java

```diff
@@ -358,7 +358,7 @@ private void writePrimitive(Object value, PrimitiveObjectInspector inspector)
                 recordConsumer.addBinary(Binary.fromString(v));
                 break;
             case CHAR:
-                String vChar = ((HiveCharObjectInspector) inspector).getPrimitiveJavaObject(value).getStrippedValue();
+                String vChar = ((HiveCharObjectInspector) inspector).getPrimitiveJavaObject(value).getPaddedValue();
```

Comment:
this is needed, since if hive writer write char without padding, and if Trino assume table's char column with padding(as in checkInBloomFilter()) the unit test would fail.